libs/core/src/components.d.ts

```diff
@@ -233,39 +233,39 @@ export namespace Components {
          */
         "componentId": string;
         /**
-          * Indicates whether or not the input field is disabled
+          * Determines whether or not the input field is disabled.
```

Comment:
The word "determines" seems to imply an automatic or programmatic setting of the value. Can we discuss a better way to describe our boolean values? I want to note that I don't think we should have a blanket rule for all boolean descriptions.

Reply:
"Determines" was decided by a previous team discussion, but if we want to revisit I'd say let's do it and then make the updates with whatever decision is decided in a follow-up ticket/pr.

Reply:
I don’t remember us having a full team discussion on this, but I do remember our brief chat about the difference between ‘Indicates’ and ‘Determines’.

We should not merge this but rather, let’s discuss the details now to ensure clarity moving forward.

Reply:
And just "whether or not" terminology before that... clarification is fine, but since this is scoped to the input component, I'd disagree with holding this up since it may eventually affect a wide variety of components.

I'd be happy to roll back the text change here and wait for the team discussion if that works better?

Reply:
It would be ideal to address the phrasing like, "If true... If false"

Reply:
@ju-Skinner updated after team convo. Check it out when time permits.